app/models/package_manager/base.rb

```diff
@@ -93,12 +93,6 @@ def self.platform_name(platform)
       find(platform).try(:formatted_name) || platform
     end
 
-    private_class_method def self.transform_mapping_values(mapping)
-      mapping.try do |p|
-        p.compact.transform_values { |v| v.is_a?(String) ? StringUtils.strip_null_bytes(v) : v }
```

Comment:
removing `compact` here is the big change -- we'll now be able to update fields like "homepage" to nil if they are nil. These will also be audited now: https://github.com/librariesio/libraries.io/pull/3438/files#diff-611e7045e8b0212d101cd856c335296959519af63b80129f51c246a7bbfe7b91R109